cmd/expose.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var ReConnect bool
```

Comment:
can we change `-r` and `reconnect` to `-p` and `persist`?

Reply:
make sense, I will update

Reply:
@omrikiei -p is conflicting with global flag `-p, --port int` can't use single char alternative for persist

may be `-r, --reuse` ?

Reply:
Sure, sorry for misleading you with that

Reply:
no problem, updated commit.